content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -125,6 +125,7 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `false` | Alpha | 1.7 | 1.9 |
 | `LocalStorageCapacityIsolation` | `true` | Beta | 1.10 | |
 | `LocalStorageCapacityIsolationFSQuotaMonitoring` | `false` | Alpha | 1.15 | |
+| `MixedProtocolLBService` | `false` | Alpha | 1.20 | |
```

Comment:
Can you provide a short description below the table on what behavior this provides?

Reply:
Yep, I added a description right now.